src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -120,12 +147,64 @@ def _detect_performance_issue(data: Event, sdk_span: Any):
         DetectorType.N_PLUS_ONE_SPANS: NPlusOneSpanDetector(detection_settings, data),
     }
 
+    # Create performance issues for duplicate spans first
+    used_perf_issue_detectors = {
+        DetectorType.DUPLICATE_SPANS_HASH: DuplicateSpanHashDetector(detection_settings, data),
+        DetectorType.DUPLICATE_SPANS: DuplicateSpanDetector(detection_settings, data),
+    }
+
     for span in spans:
         for _, detector in detectors.items():
             detector.visit_span(span)
 
     report_metrics_for_detectors(event_id, detectors, sdk_span)
 
+    detected_problems = [
+        (i, detector_type)
+        for detector_type, d in used_perf_issue_detectors.items()
+        for i in d.stored_issues
```

Comment:
nit: we should probably change `stored_issues` on the detectors as well to be `problem` to make sure all the terminology is the same